JS-Algorithms/challenges.js

```diff
@@ -13,7 +13,23 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 ***** */
 
 const readableTime = (seconds) => {
-  // YOUR CODE HERE...
+  const minutesFromSeconds = seconds / 60;
+  const hoursFromMinutes = Math.floor(minutesFromSeconds / 60);
+  const leftMinutes = Math.floor(minutesFromSeconds - (hoursFromMinutes * 60));
```

Comment:
We could use the remainder operator as well to have simpler code like:

```suggestion
  const minutes = Math.floor((seconds % 60) / 3600);
```

Reply:
Refactored ✅